tests/checkbox/data/references.csv

```diff
@@ -1,7 +1,8 @@
 refId,value
 author,Michael Fairchild
 authorEmail,michael.fairchild@deque.com
-reference,reference/2020-11-23_175030/checkbox-1/checkbox-1.html
```

Comment:
@evmiguel - There is an actual problem you seem to have lost my changes because I think you need to rebase. This needs to not change.